### PR TITLE
Fix MW 1.36 skin compatibility

### PR DESCRIPTION
**Pull request by dolfinus**
See https://github.com/dolfinus/DarkVector/issues/12 for more details

### CODE REVIEW

**Comment by dolfinus** [comment]
@jdlrobson, can you please check, are these changes correct?

**Comment by jdlrobson** [comment]
When I test out the patch the skin still doesn't render as it uses a method deprecated in 1.30. I think you need escapeIdForAttribute. The stack trace for current error is:
```
 Uncaught Error: Call to undefined method Sanitizer::escapeId() in /Users/jrobson/git/core/skins/DarkVector/DarkVectorTemplate.php:62 Stack trace: #0 /Users/jrobson/git/core/includes/skins/SkinTemplate.php(123): DarkVectorTemplate->execute() #1 /Users/jrobson/git/core/includes/skins/SkinTemplate.php(138): SkinTemplate->generateHTML() #2 /Users/jrobson/git/core/includes/OutputPage.php(2616): SkinTemplate->outputPage() #3 /Users/jrobson/git/core/includes/exception/MWExceptionRenderer.php(153): OutputPage->output() #4 /Users/jrobson/git/core/includes/exception/MWExceptionRenderer.php(65): MWExceptionRenderer::reportHTML(Object(Error)) #5 /Users/jrobson/git/core/includes/exception/MWExceptionHandler.php(106): MWExceptionRenderer::output(Object(Error), 2) #6 /Users/jrobson/git/core/includes/exception/MWExceptionHandler.php(185): MWExceptionHandler::report(Object(Error)) #7 /Users/jrobson/git/core/includes/exception/MWExceptionHandler.php(156): MWExceptionHandler::handleException(Object(Error), 'mwe_handler') #8 [in in /Users/jrobson/git/core/skins/DarkVector/DarkVectorTemplate.php on line 62
```

**Comment by jdlrobson** [comment]
With the above fix this renders correctly with one exception:

```
Deprecated: Use of BaseTemplate::getToolbox was deprecated in MediaWiki 1.35. [Called from DarkVectorTemplate::renderPortals in /Users/jrobson/git/core/skins/DarkVector/DarkVectorTemplate.php at line 274] in /Users/jrobson/git/core/includes/debug/MWDebug.php on line 329

```

**Comment by dolfinus** [comment]
Can you try now, please?

**Review by jdlrobson — CHANGES_REQUESTED** [review]
there's another one on L62

File: DarkVectorTemplate.php [hunk 1]
```diff
@@ -296,10 +296,10 @@ protected function renderPortal( $name, $content, $msg = null, $hook = null ) {
 			$msg = $name;
 		}
 		$msgObj = wfMessage( $msg );
-		$labelId = Sanitizer::escapeId( "p-$name-label" );
+		$labelId = Sanitizer::escapeIdForAttribute( "p-$name-label" );
 		?>
```
Comment (jdlrobson) [inline DarkVectorTemplate.php:300]: This is correct but see L62 for another usage.

Comment (dolfinus) [reply DarkVectorTemplate.php:300]: Fixed

**Comment by dolfinus** [comment]
Fixed

**Review by jdlrobson — APPROVED** [review]

**Pull request merged**